tests/translate/translate_moistcvpluspkz_2d.py

```diff
@@ -52,18 +56,16 @@ def __init__(self, grid):
             "q_con": {},
         }
 
-    def compute(self, inputs):
-        self.make_storage_data_input_vars(inputs)
-        inputs["j_2d"] = self.grid.global_to_local_y(
-            inputs["j_2d"] + TranslateGrid.fpy_model_index_offset
+    def compute_from_storage(self, inputs):
+        moist_cv_pkz = FrozenStencil(
+            moist_cv.moist_pkz,
+            origin=(self.grid.is_, self.grid.js, 0),
+            domain=(self.grid.nic, 1, self.grid.npz),
```

Comment:
I'm curious why the Frozen Stencil is here in compute but not in init.

Reply:
good catch! it can be in init, I'll move it there. When the init relies on serialized data it needs to happen in the compute function, but you are right, this is not needed here.

Reply:
ah, it was there because initially I was going to use j_2d to set the origin. but if we end up returning to the fortran code to write out a different j, we should probably do the step we previously decided not do, of updating serialbox to accunulate 3d variables in the j direction like it does for the k direction when k blocked. Anyways, here decided to just use the jslice value we picked to make it simpler. But then forgot to move the FrozenStencil init back to the translate init.

Reply:
Thanks for the explanation!  I was thinking maybe there was a particular reason why the stencil was in compute, so that's why I asked.